api/apps.py

```diff
@@ -1,6 +1,11 @@
+import os
 from django.apps import AppConfig
 
 
 class ApiConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'api'
+    def ready(self):
+        if os.environ.get('RUN_MAIN'):
```

Comment:
What determines this env variable?

Reply:
So the django-apschedule package that I used to automate the sender function runs at the given time interval of "24 hrs" but it calls the function twice and in other to prevent that, I have to add that line of code with the env variable which isn't set by me.
Reference: https://stackoverflow.com/questions/33814615/how-to-avoid-appconfig-ready-method-running-twice-in-django